tensorboard/webapp/plugins/npmi/npmi_component.ts

```diff
@@ -0,0 +1,26 @@
+/* Copyright 2020 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+import {ChangeDetectionStrategy, Component, Input} from '@angular/core';
+import {RunId} from '../../core/types';
+
+@Component({
+  selector: 'npmi-component',
+  templateUrl: './npmi_component.ng.html',
+  styleUrls: ['./npmi_component.css'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class NpmiComponent {
+  @Input() runs: Map<RunId, boolean> = new Map();
```

Comment:
Since there should be literally no case when we use this default `Map`, remove it (and require the `runs` to be passed with `runs!: Map<...>`).

Reply:
Done.